### PR TITLE
Make admin credentials flexible

### PR DESCRIPTION
**Pull request by SergioBertolinSG**
When adding the functionality of using big fileid numbers, the admin credentials were forced to be admin/admin. This PR solves that.

### CODE REVIEW

File: tests/integration/features/bootstrap/BasicStructure.php [hunk 1]
```diff
@@ -406,11 +407,12 @@ public static function removeFilesFromLocalStorageAfter() {
 	/**
 	 * @BeforeSuite
 	 */
-	public static function useBigFileIDs() {
+	public static function useBigFileIDs(BeforeSuiteScope $scope) {
 		$fullUrl = getenv('TEST_SERVER_URL') . "/v1.php/apps/testing/api/v1/increasefileid";
 		$client = new Client();
 		$options = [];
-		$options['auth'] = ['admin','admin'];
+		$adminCredentials = $scope->getSuite()->getSettings()['contexts'][0][__CLASS__]['admin'];
```
Comment (jvillafanez) [inline tests/integration/features/bootstrap/BasicStructure.php]: If there isn't a better option we'll have to go with this.

Comment (SergioBertolinSG) [reply tests/integration/features/bootstrap/BasicStructure.php]: Approve the review then 😋

**Review by phil-davis — APPROVED** [review]
It would be nice to be able to get the admin credentials without such odd-looking means to find the context information. But because we are in BeforeSuite here, the context is not already nicely instantiated. So this is "the best" that can be done without some bigger refactoring to do it every scenario BeforeScenario.

**Comment by phil-davis** [comment]
@SergioBertolinSG I resolved a little conflict online here. So that means CI runs again.

**Comment by SergioBertolinSG** [comment]
@phil-davis thanks.

**Comment by phil-davis** [comment]
Note: this never got backported to ``stable10``
I noticed it today when backporting #30830 I got was surprised by some little conflicts reported.

In any case, the changes in #30830 (and its backport #30842 ) have superceded the lines here. So there is no longer anything from this PR to backport.

**Pull request merged**